? gtk.window.cpp.patch
? gtk_combobox_multiple_selected_events.patch
? gtk_scroll_lineup_linedown.patch
? gtk_scroll_lineup_linedown_2.patch
? gtk_scroll_lineup_linedown_3.patch
? gtk_scroll_up_down_event_order.patch
? contrib/samples/foldbar/foldpanelbar/mondrian.xpm
? src/gtk/window.cpp.new
Index: src/gtk/scrolbar.cpp
===================================================================
RCS file: /pack/cvsroots/wxwidgets/wxWidgets/src/gtk/scrolbar.cpp,v
retrieving revision 1.54
diff -b -u -2 -r1.54 scrolbar.cpp
--- src/gtk/scrolbar.cpp	2005/01/23 18:36:43	1.54
+++ src/gtk/scrolbar.cpp	2005/02/08 15:09:55
@@ -68,8 +68,4 @@
     int orient = win->HasFlag(wxSB_VERTICAL) ? wxVERTICAL : wxHORIZONTAL;
 
-    wxScrollEvent event( command, win->GetId(), value, orient );
-    event.SetEventObject( win );
-    win->GetEventHandler()->ProcessEvent( event );
-
     // throw a LINEUP / LINEDOWN event if necessary
     if (g_currentUpDownEvent != wxEVT_NULL)
@@ -79,4 +75,9 @@
         win->GetEventHandler()->ProcessEvent( event );
       }
+    
+	// throw other event (wxEVT_SCROLL_THUMBTRACK)
+	wxScrollEvent event( command, win->GetId(), value, orient );
+    event.SetEventObject( win );
+    win->GetEventHandler()->ProcessEvent( event );
 
 /*

 	  	 
